geomstats/geometry/symmetric_matrices.py

```diff
@@ -20,3 +21,13 @@ def __init__(self, n):
     def belongs(self, mat, atol=TOLERANCE):
         """Check if mat belongs to the vector space of symmetric matrices."""
         return Matrices(self.n, self.n).is_symmetric(mat=mat, atol=atol)
+
+    def expm(x):
+        """Compute the matrix exponential."""
```

Comment:
Missing description of parameters in docstring

Reply:
No `self` nor `cls` in input parameters? Then maybe make it a static method: @staticmethod decorator.

Reply:
Oh I forgot `self`. For now, I prefer it to be a standard method.

Reply:
At some point, it could be useful to make all the classes consistent with each other for standard/static/class methods. It seems to me that, for now, the choice is a bit random. Or if there is a rule, it is not clear to me.